saleor/account/views.py

```diff
@@ -96,6 +98,15 @@ def get_or_process_password_form(request):
     return form
 
 
+def get_or_process_name_form(request):
+    form = NameForm(data=request.POST or None, instance=request.user)
+    if form.is_valid():
+        form.save()
+        messages.success(request, pgettext(
+            'Storefront message', 'Data successfully changed.'))
```

Comment:
We know what data was changed and we could give a more user-friendly feedback. What do you think about something like: "Account successfully updated"?